client/client-multi/client-c/CMakeLists.txt

```diff
@@ -201,11 +201,19 @@ set(KAA_INCLUDE_DIRS
         ${KAA_SRC_FOLDER}
         ${KAA_THIRDPARTY_INCLUDE_DIR})
 
+# Generated include directory
+set(GENERATED_INCLUDE_DIR ${PROJECT_BINARY_DIR}/gen_includes)
```

Comment:
Why not just use `${CMAKE_CURRENT_BINARY_DIR}/kaa_keys/`? Then we can drop "Copying generated files" step.